modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -236,6 +236,8 @@ apiTemplateFiles are for API outputs only (controllers/handlers).
     // flag to indicate whether enum value prefixes are removed
     protected boolean removeEnumValuePrefix = true;
 
+    // set to true to promote inline enums to their own separate models
+    protected boolean resolveInlineEnums = false;
```

Comment:
We also have the ModelUtils.unaliasScema that decides whether or not a model is made.
How about naming this resolveInlineSchemasAsModels.
Doesn't it apply to more than just enums?

Reply:
The specific request from previous feedback was to treat enums differently here than other models, this flag actually only applies to enums not other models as the name implies. See: https://github.com/OpenAPITools/openapi-generator/pull/3162#issuecomment-520193930